docs/proposals/002-api-proposal/proposal.md

```diff
@@ -0,0 +1,233 @@
+
+The Inference Platform Admin creates and manages the infrastructure necessary to run LLM workloads. Including handling Ops for: 
+  - Hardware
+  - Model Server
+  - Base Model
+  - Resource Allocation for Workloads
+  - Gateway configuration
+  - etc
+
+#### LLM Use Case Owner
+
+An LLM Use Case Owner persona owns and manages 1 or many Generative AI Workloads (LLM focused *currently*). This includes:
+- Defining SLO
+- Deploying LoRA Adapters (or other fine-tune)
+- Managing rollout of adapters
+
+### Axioms 
+
+The API design is based on these axioms:
+
+- Pools of shared compute should be *discrete* for scheduling to properly work
+- Pod-level scheduling should not be handled by a high-level gateway 
+- Simple use cases should be simple to define (or are implicitly defined via reasonable defaults)
+- This solution should be composable with other Gateway solutions and flexible to fit customer needs
+- The MVP will heavily assume requests are done using the OpenAI spec, but open to extension in the future
+
+The [PoC](https://youtu.be/NUBZg_uqqXk?si=v681EeYdGUGEVqQQ&t=1458) was focused on lower-level scheduling. And the API follows that similar logic, which lead to the proposal of the **BackendPool**.
+
+### BackendPool
+
+*** FOR MVP THE BACKEND IS PROPOSED TO BE IMPLICIT ***
+
+The BackendPool at its core is a logical grouping of compute, expressed in the form of Pods (typically model servers), akin to a K8s Service. The BackendPool would deploy its own routing, and offer administrative configuration to the Platform Admin. 
+
+ It is expected for the BackendPool to:
+ - Enforce fair consumption of resources across competing use cases
+ - Efficiently route requests across shared compute (as displayed by the PoC)
+ 
+It is _not_ expected for the BackendPool to:
+ - Enforce any common set of adapters or base models are available on the Pods
+ - Manage Deployments of Pods within the Pool
+ - Manage Pod lifecycle of pods within the pool 
+
+Additionally, any Pod that seeks to join a BackendPool would need to support a protocol, defined by LLM Instance Gateway, to ensure the Pool has adequate information to intelligently route requests.
+
+### LLMUseCase
+
+An LLMUseCase allows the UseCaseOwner to define:
+- Which LoRA adapter(s) to consume 
+  - LLMUseCase allows for traffic splitting between adapters _in the same backendpool_ to allow for new LoRA adapter versions to be easily rolled out 
+- SLO objectives for the UseCase
+- The Pools this UseCase is relevant to 
+
+### Spec
+
+**LLMUseCase**
+```golang
+// LLMUseCaseSet represents a set of LLM use cases that are multiplexed onto one or more backend pools.
+// This is generally owned by the "LLM Use Case Owner" persona, which can be teams in an organization.
+type LLMUseCaseSet struct {
+        metav1.ObjectMeta
+        metav1.TypeMeta
+
+        Spec LLMUseCaseSetSpec
+}
+
+type LLMUseCaseSetSpec struct {
+        // Defines the use cases in the set.
+        UseCases   []LLMUseCase
```

Comment:
Define how multiple use cases from multiple top level objects compose (priority).

Reply:
Added a little more detail. Attempting to rely on the glossary supplemental doc to define what priority does to centralize definitions of terms. LMK if this lands for you or if we should have more details.